app/segments/drill/[id]/submission/input.js

```diff
@@ -829,6 +827,9 @@ const styles = StyleSheet.create({
     alignItems: "center",
     height: "contain",
     justifyContent: "center",
+    position: "absolute",
+    bottom: 25,
+    width: "100%",
```

Comment:
I tried this approach at one point in development. It works for most things but there is a glaring bug on android when the keyboard is brought up. One thing I tried to do was hiding the nav section when there was a focus on an input but it would glitch when a you would click from one input to the next. so we will have to explore another solution. 
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/c9226f46-e773-4411-89f2-bb1c20837fe0)

Reply:
I've just pushed a fix, see if it's good now